invoices_service.go

```diff
@@ -216,3 +217,30 @@ func (s *invoicesImpl) RefundVoidOpenAmount(invoiceNumber int, amountInCents int
 
 	return resp, &dst, err
 }
+
+// RecordPayment records an offline payment for a manual invoice.
+// https://dev.recurly.com/v2.5/docs/enter-an-offline-payment-for-a-manual-invoice-beta
+func (s *invoicesImpl) RecordPayment(offlinePayment OfflinePayment) (*Response, *Transaction, error) {
+	action := fmt.Sprintf("invoices/%d/transactions", offlinePayment.InvoiceNumber)
+	data := struct {
```

Comment:
Why not use the `xml` struct tags already in `OfflinePayment`? Looks like it's just missing the `XMLName` field and the `omitempty` flags.